syntax = "proto3";

package sample;

message Born {
  string date = 1;
  string city = 2;
}

message Car {
  string model_name = 1;
  int32 factory_year = 2;
  string license_plate = 3;
}

message Home {
  string address = 1;
  string ownership = 2;
}

message PersonFirst {
  string name = 1;
  Born born = 2;
  Car car = 3;
  Home home = 4;
  string job = 5;
}

message PeopleFirst {
  repeated PersonFirst people = 1;
}

